dnf/cli/output.py

```diff
@@ -415,7 +411,7 @@ def simpleEnvraList(self, pkg, ui_overflow=False,
             columns = (-63, -16) # Old default
         envra = '%s%s' % (indent, ucd(pkg))
         hi_cols = [highlight, 'normal', 'normal']
-        rid = pkg.ui_from_repo
+        rid = pkg.reponame
```

Comment:
You are right, this should be `ui_from_repo` - I guess that this is dnf.Package, SwdbPkg doesn't even have attribute `reponame`.